cloud_platform/models/cloud_platform.py

```diff
@@ -156,7 +156,10 @@ def _check_s3(self, environment_name):
                 "AWS_SECRET_ACCESS_KEY environment variable is required when "
                 "ir_attachment.location is 's3'."
             )
-            bucket_name = os.environ.get('AWS_BUCKETNAME')
+            bucket_name = (
+                os.environ.get('AWS_BUCKETNAME') or
+                os.environ.get('ODOO_DEV_KEEP_ATTACHMENT_S3')
```

Comment:
To be consistent, we need to use a common variable for S3 and SWIFT.
The modification here must be added also into function `_check_swift` just above this function.
And the name of environment variable must be changed into odoo-template: https://github.com/camptocamp/odoo-template/blob/stable/%7B%7Bcookiecutter.repo_name%7D%7D/odoo/before-migrate-entrypoint.d/000_dev_set_db_storage#L9